libs/build-nss-desktop.sh

```diff
@@ -15,6 +15,8 @@ NSS_SRC_DIR=${1}
 # Whether to cross compile from Linux to a different target.  Really
 # only intended for automation.
 CROSS_COMPILE_TARGET=${2-}
+# This will stay consistent with CARGO_CFG_TARGET_ARCH
```

Comment:
Hate to make more nits, but this comment might leave the impression `CARGO_CFG_TARGET_ARCH` can be `unknown` - so a bit more explicit here makes sense `We only need this in a couple of places - when it's not "unknown" it will be ...etc`

Reply:
all good, definitely want to make sure we bake as much context as possible so it's easy to pick-up/understand in the future!